libraries/api/include/golos/api/reblog_entry.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include <golos/chain/database.hpp>
+
+namespace golos { namespace api {
+
+    struct reblog_entry {
+        std::string title;
+        std::string body;
+        std::string json_metadata;
+
+        reblog_entry() {
+        }
+
+        reblog_entry(const std::string& title_, const std::string& body_, const std::string& json_metadata_)
```

Comment:
May be initialize from blog_object? It will require less copy-paste at calling places.

Reply:
@afalaleev blog_object is in plugin, reblog_entry is in api-library (couldn't be moved to plugin because used in discussion class).
Also, get_feed and get_feed_entries are initially big copy-paste, and it isn't single case.